las_geoh5/import_las.py

```diff
@@ -299,14 +314,15 @@ def create_or_append_drillhole(
     return drillhole
 
 
-def las_to_drillhole(
+def las_to_drillhole(  # pylint: disable=too-many-arguments
     workspace: Workspace,
     data: lasio.LASFile | list[lasio.LASFile],
     drillhole_group: DrillholeGroup,
     property_group: str | None = None,
     survey: Path | list[Path] | None = None,
     translator: LASTranslator | None = None,
-) -> Drillhole:
+    skip_empty_header: bool = False,

```

Comment:
**suggestion (doc):** add doc for `translator` and `skip_empty_header`

Reply:
done.